cmd/miniooni/main.go

```diff
@@ -273,7 +275,11 @@ func main() {
 		defer experiment.CloseReport(ctx)
 	}
 
+	step := 100.00 / float64(len(inputs))
```

Comment:
Here you only have weak guarantees that `inputs` is not empty. Please add a comment saying that we know it is not empty because the code above ensures that this is the case.